README.md

```diff
@@ -206,6 +206,12 @@ The filtering can also be applied for pins with `subscribe` attribute.
 
 ## Changelog
 
+### v4.2.0
+
+#### Feature:
+
+* In case of decoding error instead of skipping group replace raw messages of empty or target protocol with `th2-codec-error` message in them```
```

Comment:
```suggestion
* In case of decoding error, instead of skipping the group, replace raw messages of empty or target protocol with `th2-codec-error` message in them
```